test_two_mgr_pods_and_metadata failed in external mode cluster

Test run: https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/632/22405/1070590/1070593/log 
```
  assert len(mgr_pods) == 2, "There should be 2 mgr pods"
E       AssertionError: There should be 2 mgr pods
E       assert 0 == 2
E         +0
E         -2

tests/functional/pod_and_daemons/test_mgr_pods.py:46: AssertionError
```

--- a/tests/functional/pod_and_daemons/test_mgr_pods.py
+++ b/tests/functional/pod_and_daemons/test_mgr_pods.py
@@ -7,6 +7,7 @@ from ocs_ci.framework.testlib import (
     post_upgrade,
     post_ocs_upgrade,
     polarion_id,
+    skipif_external_mode,
 )
 from ocs_ci.ocs import exceptions
 from ocs_ci.ocs.resources import pod
@@ -17,6 +18,7 @@ log = logging.getLogger(__name__)
 
 @tier4a
 @skipif_ocs_version("<4.15")
+@skipif_external_mode
 @post_upgrade
 @post_ocs_upgrade
 class TestMgrPods(BaseTest):

